Gems/Atom/Feature/Common/Code/Source/PostProcessing/HDRColorGradingPass.cpp

```diff
@@ -128,7 +128,7 @@
             if (scene)
             {
                 PostProcessFeatureProcessor* fp = scene->GetFeatureProcessor<PostProcessFeatureProcessor>();
-                AZ::RPI::ViewPtr view = scene->GetDefaultRenderPipeline()->GetDefaultView();
+                AZ::RPI::ViewPtr view = scene->GetDefaultRenderPipeline()->GetFirstView(GetPipelineViewTag());
```

Comment:
Maybe irrelevant to your PR, it probably shouldn't use scene->GetDefaultRenderPipeline() but use m_pipeline (or GetRenderPipeline()